lib/travis/api/serialize/v2/http/job.rb

```diff
@@ -43,9 +43,10 @@ def job_data
                   'allow_failure' => job.allow_failure,
                   'tags' => job.tags,
                   'annotation_ids' => job.annotation_ids,
-                }.tap do |ret|
-                  ret['log_id'] = job.log_id if include_log_id?
-                end
+                }
+                data['log_id'] = job.log_id if include_log_id?
+                data['stage_id'] = job.stage_id if job.stage_id
```

Comment:
I think it's more consistent to always return the stage_id as part of the return object even if it's nil.